httpcore/src/main/java/org/apache/http/impl/EnglishReasonPhraseCatalog.java

```diff
@@ -140,6 +140,8 @@ private static void setReason(final int status, final String reason) {
                   "Forbidden");
         setReason(HttpStatus.SC_NOT_FOUND,
                   "Not Found");
+        setReason(HttpStatus.SC_TOO_MANY_REQUESTS,
+                "Too Many Requests");
```

Comment:
Nit: The indentation does not match.

Reply:
@garydgregory I've fixed the indentation